jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -0,0 +1,61 @@
+job {
+  name 'terraform-deploy'
+  description('Deploy and automatically update Terraform environments upon repository changes')
+  parameters {
+    choiceParam("action", ["apply", "destroy"], "Select if you want to apply changes to environments or destroy them.")
```

Comment:
Soft wrapping can make it harder to read, in my opinion, especially in formats like YAML where whitespace is significant. Not everybody's editor is the same and we have no control over some things like GitHub, so let's optimise for that and change this, please.